src/STW.SubmissionApi.Presentation/Services/QueueService.cs

```diff
@@ -0,0 +1,30 @@
+using Azure.Messaging.ServiceBus;
+using Microsoft.AspNetCore.Mvc;
+
+namespace STW.SubmissionApi.Presentation.Services;
+
+public class QueueService
+{
+    public async Task Send(string queueName, string requestBody)
+    {
+        ServiceBusClientOptions clientOptions = new ServiceBusClientOptions()
```

Comment:
Some of these configurations can be moved out of the class to configuration extensions. We can also make use of dependency injection rather than new.

You should also program to an interface, not a concrete class. This makes changing out underlying technologies in future easier should we move away from queues. In this case, I would consider renaming the class to `MessagingService` rather than `QueueService`.

Reply:
You also need unit tests for this class

Reply:
Updated